package.json

```diff
@@ -12,16 +12,13 @@
   "module": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "exports": {
-    ".": {
-      "require": "./dist/index.cjs",
-      "import": "./dist/index.js"
-    },
-    "./types": "./dist/index.d.ts",
-    "./mockResponses/*": "./dist/mockResponses/*"
+    "require": "./dist/index.cjs",
+    "import": "./dist/index.js",
+    "types": "./dist/index.d.ts"
```

Comment:
I was really trying to get the mock responses to export from `tincheck/mock-responses`

I had it working as far as being able to import the files running in our dev. But eslint was really struggling to resolve the types without updating the main repo module resolution

Reply:
The issue with this approach (I tried this) is it bundles all the files in dist/index.js so the bundle size increases drastically. What you could try is running `tsup` to compile mockResponses.ts file in a sequential build step

Reply:
I was doing that
I'll maybe push what I had, if you like it we can publish

Maybe if it's through npm the resolution will work. With linking it wouldn't